app/views/maintenance_tasks/tasks/show.html.erb

```diff
@@ -16,6 +16,16 @@
           <%= form.file_field :csv_file %>
         </div>
       <% end %>
+      <% if @task.parameter_names.any? %>
+        <div class="block">
+          <%= form.fields_for :task_arguments do |ff| %>
+            <% @task.parameter_names.each do |parameter| %>
+              <%= ff.label parameter, "#{parameter}: ", class: "label" %>
+              <%= ff.text_area parameter %>
```

Comment:
It's barely visible with Firefox. Doesn't it need a class like everything does with Bulma?

Reply:
Yeah, forgot to circle back on this. I'll add Bulma's `textarea` class and see if that works for FF.